doc/api/v0.yaml

```
@@ -441,3 +464,40 @@ components:
               type: array
               items:
                 $ref: "#/components/schemas/Proposal"
+
+    ReviewTag:
+      type: string
+      enum:
+        - Alignment
+        - Verifiability
+        - Feasibility
+        - Impact
+        - Auditability
+
+    Review:
```

Let's make the names consistent: `AdvisorReview` and its plural `AdvisorReviews`.
```suggestion
    AdvisorReview:
```